<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
      <item>
       <widget class="GLYUVRender" name="widget_screen" native="true"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
          <item>
           <widget class="QSlider" name="Slider">
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>音量</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="Slider_Volume">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_controlButtons">
          <item>
           <widget class="QPushButton" name="pushButton_openFile">
            <property name="text">
             <string>打开</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_playPause">
            <property name="text">
             <string>播放</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_stop">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_screen">
            <property name="text">
             <string>全屏</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_speedUp">
            <property name="text">
             <string>加速</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_speed">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>1.0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_speedDown">
            <property name="text">
             <string>减速</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_hardware">
            <property name="text">
             <string>软/硬解</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_playList">
            <property name="text">
             <string>播放列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_UseIODevice">
            <property name="text">
             <string>流播放</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLYUVRender</class>
   <extends>QWidget</extends>
   <header>glyuvrender.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
